extensions/wikia/Search/classes/Test/ResultTest.php

```diff
@@ -276,82 +276,80 @@ public function testVarMethods() {
 	 * @covers Wikia\Search\Result::fixSnippeting
 	 */
 	public function testFixSnippeting() {
-
-		$result		= new Result( $this->defaultFields );
 		$method		= new ReflectionMethod( 'Wikia\Search\Result', 'fixSnippeting' );
 		$method->setAccessible( true );
 
 		$text = '�foo';
 		$this->assertEquals(
 			'foo',
-			$method->invoke( $result, $text ),
+			$method->invoke( null, $text ),
 			'Wikia\Search\Result::fixSnippeting should remove bytecode junk.'
 		);
 
 		$text = 'foo &hellip;';
 		$this->assertEquals(
 			'foo',
-			$method->invoke( $result, $text ),
+			$method->invoke( null, $text ),
 			'Wikia\Search\Result::fixSnippeting should remove string-final ellipses.'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Wikia\Search\Result::fixSnippeting should remove string-final ellipses." 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)